docs/api-guide.md

```diff
@@ -0,0 +1,722 @@
+# Java API Guide
+
+This guide briefly explains how to use Scalar DB Java API.
```

Comment:
Sorry, it might be better to say something like this since showing the bullet points is a bit sudden. 
```suggestion
Scalar DB Java API is mainly composed of Administrative API and Transactional API. This guide briefly explains what kind of APIs exist and how to use them.
```